File: deskNG/src/app/desk-manager/models/request-task.ts
import { Message } from './message';

export class RequestTask{
    constructor(
        public id: number,
        public theme: string,
        public user: string,
        public admin: string,
        public status: string,
        public shop: string,
        public phone: string,
        public dateOpen: string,
        public dateClose: string,
        public messages: Array<Message>,
    ){}
}

File: DeskApiManager/DeskApiManager/Models/RequestTask.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DeskApiManager.Models
{
    public class RequestTask
    {
        public int Id { get; set; }
        public string Theme { get; set; }
        public string User { get; set; }
        public string Admin { get; set; }
        public string Status { get; set; }
        public int? ObjectId { get; set; }
        public int? DepartmentId { get; set; }
        public string Phone { get; set; }
        public string DateOpen { get; set; }
        public string DateClose { get; set; }
        public ICollection<Message> Messages { get; set; }
    }
}
